diagnostic.md

```diff
@@ -49,7 +55,9 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@@function tint($cornflower, 20%) {
+  @return mix ($pure-white, $cornflower, 80%;);
+}
 ```
```

Comment:
Good work with the idea of a SASS function, but here we just needed you to call upon the tint function, same as we worked with in class yesterday like so:

`tint($cornflower, 20%);`